torchaudio/backend/common.py

```diff
@@ -1,8 +1,13 @@
 from typing import Any, Optional
 
+from torchaudio._internal import module_utils as _mod_utils
+
 
 class AudioMetaData:
-    """Data class to be returned by :py:func:`~torchaudio.info`.
+    """Data class to be returned by ``torchaudio.info`` function.
```

Comment:
This is a little misleading because AudioMetaData isn't actually a Python dataclass, right? 

This could just say "Return type of torchaudio.info."